src/connect-zbt-1/about-the-device/about-connect-zbt-1.md

```diff
@@ -0,0 +1,49 @@
+---
+zendesk:
+  article_id: 30878291112349
+  name: About Home Assistant Connect ZBT-1
+  position: 1
+  labels: connect zbt-1, about the device
+---
+
+[Home Assistant Connect ZBT-1](https://www.home-assistant.io/connectzbt1) is a Zigbee and Thread adapter. It was designed specifically to work with Home Assistant. It connects to your Home Assistant system via USB, allowing it to build either a Zigbee or a Thread network via Home Assistant's built-in Zigbee or Thread integrations.
+
+{% callout "important" %}
+
+Home Assistant Connect ZBT-1 is not a standalone product.
+
+- It requires Home Assistant installed on a smart home hub (such as [Home Assistant Green](https://www.home-assistant.io/green/)).
+- It requires Zigbee or Thread end devices to form a network.
```

Comment:
nit/optional:
Technically, the network can be formed without any Zigbee/Thread devices, but this might be ok to say for clarity. Or is this referencing a Zigbee or Thread adapter (needed to form the network?

Also, I'm not sure if "Zigbee or Thread **end** devices" is really the correct term, regardless of what we're talking about. In the Zigbee world, "end-devices" are technically mostly battery-powered devices. For Thread, I think this is kind of similar?
But if you think it's clearer to use "end devices", it should be fine, even if it's not fully accurate in a technical sense, I guess.